### DISCUSSION TITLE
Can we use nullptr instead of NULL?

### DISCUSSION THREAD

**Discussion by sairam4123** [Drachenfrucht1/godotcord#64] [Q&A]
I hope it's better to use nullptr instead of NULL what do you think?

**Comment by jordigulley** [comment]
As far as I've seen, you should use nullptr, since NULL is just 0 (int). nullptr is more explicit.

![image](https://user-images.githubusercontent.com/12436824/115124157-6ee70500-9f86-11eb-9aa9-ccaad5ce1bcf.png)

**Reply by sairam4123** [reply]
Godot 4 uses nullptr, so we can use it.
